src/layer-composer/types.ts

```diff
@@ -33,3 +33,8 @@ export interface GeneratorConfig {
   visible?: boolean
   opacity?: number
 }
+
+export interface GlobalGeneratorConfig {
+  start?: string
```

Comment:
I've been doing some research to ensure this is an isoString but typescript doesn't allow it... only checks types... 
so what about using something like this https://github.com/typestack/class-validator to ensure this uses the desired format (https://github.com/typestack/class-validator/blob/90a663866eaeb5a0cb38cce5501e0aa28984ed34/src/validation/Validator.ts#L401)

Reply:
I like the idea but I'm not sure how this can apply to TS interfaces?